### DISCUSSION TITLE
Roundtype/Gamemode ideas

### DISCUSSION THREAD

**Discussion by Mannybrado** [shiptest-ss13/Shiptest#339] [Ideasguy]
Post roundtype ideas here. Can be anything from massive sector-wide events to small, unique conflicts of interest. I will begin;

**Unexplored Sector**
The standard shipping and testing experience, our current practical "roundtype". Explore, experience liver failure in the Boyardee, die horribly to buffed fauna tendrils, get corpse-deleted when another ship docks and undocks from your landing site. 

**Conflict Zone**
A twist on the existing setup, based on inciting a central high-power conflict. Spawns two heavy battleship-class vessels(unique to this gamemode) and potentially some combat/support escorts, based on roundpop. The two involved factions are encouraged to do what large, opposed groups of people with large guns do best, while independent vessels treat wounded (finally, an actual field day for medical ships) sell resources to either side, try to live normally within the conflict, or even join as mercenaries/spies/scavengers. 

**Abberant Region**
WIP Idea: A high danger, high reward region containing "infected" versions of normal planet types, filled with some kind of blob/hivemind esque corruption of their normal contents and rare loot. The infection can spread to players, forming an emergent conversion antag. 
Possible caveats: Some kind of infection opposing faction vessel exists and is actively trying to clean it out, there's some kind of central frontier station/settlement uninfected players must try to keep the infection from overrunning.

**Resource Rush**
A new, highly valuable resource has been spotted in asteroids around or on a planet within the sector. Set up deep drills in specific locations on the asteroid/planet to compete to mine what little of it there is, and gain access to unique tools/constructions or sell it at the station(which is guaranteed to spawn) for a pissload of cash. 
A possible extension or variation if its not interesting enough; A mining company that starts with upgraded mining gear with the explicit objective to get as much as the resource as possible, and a SMALL pirate ship is spawned to pick at them(or anyone).

**Cult Rising**
Cult, except reworked. Instead of running around capturing people, they instead run around capturing planets(this would be a process of constructing one or two big monuments,,, that can be destroyed. once built, they could be teleported to freely for easier defense, however) in order to create a network and summon their respective god. Blood cult would be explicitly aggressive, highly visible, and fairly strong, BUT, a para-normal strike ship would be deployed to combat them. Clock cult, however, would be passive, weaker, but not nearly as noticeable, and could do everything they need to do without killing anyone-- also, the strike team wouldn't be deployed until after they summoned their god, which would be an entity on the overmap that could potentially be defeated after the fact. (ideasgu)

**Dry Sector**
On this round, the sector has significantly less asteroids around and there is only one planet to get things from. Also, if there is a station, its normal resources are limited. Perhaps this could be layered on top of any other existing gamemode for the funny.

**Faction-Controlled Sector**
On this round, the sector is controlled by a faction. The station is specifically aligned to them, and the starter ship is always one of theirs. Also, all of their ships for this round are FREE.

**Comment by ZephyrTFA** [comment]
Use the hack.md

**Reply by Mannybrado** [reply]
i was told to make a discussion 
guh!
